src/components/SearchForm/SearchForm.js

```diff
@@ -23,6 +36,7 @@ export default function SearchForm({ onSearch, onLoading, loadingStatus: isLoadi
             <p className="search__subtitle">Find the latest news on any topic and save them in your personal account.</p>
             <div className="searchbar">
                 <input type="search" className="searchbar__input" placeholder="Enter topic" onChange={(e) => setQuery(e.target.value)}></input>
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components